spark-3.3/clickhouse-spark/src/main/scala/org/apache/spark/sql/clickhouse/ClickHouseSQLConf.scala

```diff
@@ -209,4 +209,13 @@ object ClickHouseSQLConf {
       .stringConf
       .transform(_.toLowerCase)
       .createWithDefault("binary")
+
+  val READ_SETTINGS: OptionalConfigEntry[String] =
+    buildConf("spark.clickhouse.read.settings")
+      .doc("Settings when read from ClickHouse. e.g. `final=1, max_execution_time=5`")
+      .version("0.9.0")
+      .stringConf
+      .transform(_.toLowerCase)
```

Comment:
why should convert the input to lower case?

Reply:
in case we'd like to log [scanQuery](https://github.com/ClickHouse/spark-clickhouse-connector/pull/367/files#diff-28e03468572c13766fd5ea52624f944c10886f6781d262bcfbda36362f1fdda0L49) which keeps SQL keywords in uppercase and other text in lowercase